tests/Unit/Subscriber/Behavior/Mapping/LocaleSubscriberTest.php

```diff
@@ -22,15 +23,18 @@ class LocaleSubscriberTest extends \PHPUnit_Framework_TestCase
 {
     public function setUp()
     {
-        $this->hydrateEvent = $this->prophesize(HydrateEvent::class);
         $this->notImplementing = new \stdClass();
         $this->node = $this->prophesize(NodeInterface::class);
         $this->document = new TestLocaleDocument();
         $this->accessor = new DocumentAccessor($this->document);
         $this->registry = $this->prophesize(DocumentRegistry::class);
+        $this->manager = $this->prophesize(DocumentManagerInterface::class);
```

Comment:
When removed from the constructor below this can also be removed. And also this file is missing all member variable declarations.